rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2319,6 +2319,36 @@ void voteFederationChange_commitEmptyFederation_returnsInsufficientMembersRespon
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorPublicKeyWithLessThanMofNVotes_returnsPendingFederationSizeZeroResponseCode() {
+
+            // Arrange
+
+            BtcECKey expectedBtcECKey = new BtcECKey();
```

Comment:
Let's ose BitcoinTestUtils to create a deterministic key

Reply:
Updated.